src/components/Square.js

```diff
@@ -5,7 +5,7 @@ const Square = ({ id, className, space, name, stamp, flagPath }) => {
     return (
         <div className={className} onClick={() => stamp(id, space)}>
             <p className="country-name">{name}</p>
-            <img className="flag-image" src={flagPath} />
+            {flagPath &&  <img className="flag-image" alt={`Flag of ${name}`} src={flagPath} />}
```

Comment:
I do not understand why this conditional is necessary, but it does appear to be working